codegenerator/cli/templates/static/codegen/src/eventFetching/FetchState.res

```diff
@@ -785,26 +772,20 @@ let getEarliestEvent = (self: t) => {
   }
 }
 
-let makeInternal = (
-  ~registerType,
+/**
+Instantiates a fetch state with root register
+*/
+let make = (
   ~staticContracts,
   ~dynamicContractRegistrations: array<TablesStatic.DynamicContractRegistry.t>,
   ~startBlock,
+  ~endBlock,
   ~isFetchingAtHead,
-  ~logger,
+  ~logger as _,
 ): t => {
   let contractAddressMapping = ContractAddressingMap.make()
 
   staticContracts->Belt.Array.forEach(((contractName, address)) => {
-    Logging.childTrace(
-      logger,
-      {
-        "msg": "adding contract address",
-        "contractName": contractName,
-        "address": address,
-      },
-    )
```

Comment:
Should I keep the log? I don't see a big use in it

Reply:
No I don't think so.